test/teiserver_web/tachyon/matchmaking_test.exs

```diff
@@ -59,8 +118,64 @@ defmodule Teiserver.Tachyon.MatchmakingTest do
     {:ok, queue_id: attrs.id, queue_pid: pid}
   end
 
-  defp setup_queue(_context) do
-    mk_queue(1)
+  describe "list" do
+    setup [{Tachyon, :setup_client}, :setup_queue]
+
+    test "works", %{client: client, queue_id: q1v1_id} do
+      {:ok, queue_id: q2v2_id, queue_pid: _} = setup_queue(2)
+
+      resp = Tachyon.list_queues!(client)
+
+      # Convert into a set since the order must not impact test result
+      expected_playlists =
+        MapSet.new([
+          %{
+            "id" => q1v1_id,
+            "name" => q1v1_id,
+            "numOfTeams" => 2,
+            "teamSize" => 1,
+            "ranked" => true,
+            "engines" => engine_names(),
+            "games" => game_names(),
+            "maps" => map_names(q1v1_id)
+          },
+          %{
+            "id" => q2v2_id,
+            "name" => q2v2_id,
+            "numOfTeams" => 2,
+            "teamSize" => 2,
+            "ranked" => true,
+            "engines" => engine_names(),
+            "games" => game_names(),
+            "maps" => map_names(q2v2_id)
+          }
+        ])
+
+      # Checking for subset because the response also contains default queues
+      assert MapSet.subset?(expected_playlists, MapSet.new(resp["data"]["playlists"]))
```

Comment:
Alternatively you can use [Enum.member?/2](https://hexdocs.pm/elixir/Enum.html#member?/2) for each map you're expecting.